ascent/configs/hydra/launcher/alliancecan.yaml

```
@@ -0,0 +1,14 @@
+defaults:
+  - submitit_slurm
+
+timeout_min: ${oc.select:run_time_min,60}
+setup:
+  - "module load httpproxy" # load module allowing to connect to whitelisted domains
+  - "source ${venv}/bin/activate" # activate the pre-installed virtual environment
+  - "mkdir $SLURM_TMPDIR/ASCENT/data -p" # make sure the full path to where the data will be copied exists
+  - "rsync -a ${paths.data_dir}/ $SLURM_TMPDIR/ASCENT/data" # copy the dataset to the compute node
```

```suggestion
  - "rsync -a ${paths.data_dir}/${datamodule.dataset_name} $SLURM_TMPDIR/ASCENT/data" # copy the dataset to the compute node
```
It might be better to just copy the specific dataset instead of all data.

---

Oh, absolutely! I just wasn't familiar enough with ASCENT to think that the name of the folder for the specific dataset was so easily accessible :sweat_smile: